x-pack/plugins/apm/server/routes/services/route.ts

```diff
@@ -129,18 +130,23 @@ const servicesRoute = createApmServerRoute({
         : Promise.resolve(null),
       getRandomSampler({ security, request, probability }),
     ]);
-    const searchAggregatedTransactions = await getSearchAggregatedTransactions({
-      ...setup,
-      kuery,
-      start,
-      end,
-    });
+
+    const { apmEventClient, config } = setup;
+    const { searchAggregatedTransactions, searchAggregatedServiceMetrics } =
+      await getServiceInventorySearchSource({
```

Comment:
shouldn't environment be here too?

Reply:
looking closer to this! 

I think, if we want to be 100% accurate environment should be there, otherwise we ask if there is ANY document that matches the `metricset.name: service` and it might no be for the given environment. 

but I'm wondering how is that different from `getSearchAggregatedTransactions`, why we don't include it as a filter for the aggregated transactions?

Reply:
it should be there too. Not sure why it isn't, it may have been on `setup` before.

Reply:
I can create a ticket it unless you want to